recipes/chromeister/meta.yaml

```diff
@@ -16,14 +16,38 @@ requirements:
   build:
     - {{ compiler('c') }}
     - make
+    - python >=3
+    - pip
+    - scikit-build
+    - cycler
+    - kiwisolver
+    - matplotlib-base
+    - numpy
+    - opencv
+    - Pillow
+    - pyparsing
+    - python-dateutil
+    - six
   host:
   run:
     - r-ape
     - r-base
+    - python >=3
+    - scikit-build
+    - cycler
+    - kiwisolver
+    - matplotlib-base
+    - numpy
+    - opencv
+    - Pillow
+    - pyparsing
+    - python-dateutil
+    - six
 
 test:
   commands:
     - CHROMEISTER --help 2>&1 > /dev/null
+    - python detect_events.py > /dev/null
```

Comment:
You should make it a "real executable":

- add `#!/usr/bin/env python` as first line in the script
- make it executable `chmod ugo+x detect_events.py`
- call it as `detect_events.py`

As a side note the: python script has some unused matplotlib imports which can be removed... are all the new requirements needed here?

Reply:
I guess most of the requirements are only needed in the run section, if at all.

Reply:
Thanks a lot for your help @bernt-matthias !

I just made it executable and removed matplotlib, thanks for letting me know

Reply:
> I guess most of the requirements are only needed in the run section, if at all.

OK, let me see if the tests work now and then I remove them